jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/WatchImageConfiguration.java

```diff
@@ -11,7 +11,7 @@
  * Contributors:
  *   Red Hat, Inc. - initial API and implementation
  */
-package org.eclipse.jkube.kit.build.service.docker.config;
+package org.eclipse.jkube.kit.config.image;
```

Comment:
Since WatchMode enum constants are now UPPER CASE

line 52 should be changed Adding a test for this builder should also be done in order to avoid future problems

```
final WatchImageConfiguration result = WatchImageConfiguration.builder().watchMode("bUiLd").build();
assertThat(result.getMode(), is(WatchMode.BUILD));
```

Reply:
Anyway, this enum constant is still used (nested) in Model configurations that can be deserialized (ImageConfiguration>WatchImageConfiugration). I strongly recommend to keep them lower case until we figure out a way to deserialize these value using some sort of case insensitive method (probably using a manual setter and constructor).